commit-graph.c

```diff
@@ -776,10 +781,16 @@ static void fill_commit_graph_info(struct commit *item, struct commit_graph *g,
 	date_low = get_be32(commit_data + g->hash_len + 12);
 	item->date = (timestamp_t)((date_high << 32) | date_low);
 
+	/* Read topological level from CDAT */
+	graph_data->level = get_be32(commit_data + g->hash_len + 8) >> 2;
```

Comment:
I don't think we will want `level` at all if we have generation v2, right?

Reply:
But, if we are persisting it across the files, then I understand why you want to keep it in the struct.